posthog/templates/email/fatal_plugin_error.html

```diff
@@ -1,25 +1,25 @@
 {% extends "email/base.html" %} {% load posthog_assets %} {% load posthog_filters %}
-{% block heading %}Fatal app error{% endblock %}
+{% block heading %}A PostHog app has experience a fatal error{% endblock %}
 {% block section %}
 <!-- prettier-ignore -->
 <p>
     {% if is_system_error %}
-        A system error has prevented {{ plugin }}
+        Uh-oh. A system error prevented {{ plugin }}
     {% else %}
-        An unhandled error in the <code>setupPlugin</code> function of {{ plugin }} has prevented it
-    {% endif %} from loading in project {{ team }}:
+        Uh-oh. An unhandled error in the <code>setupPlugin</code> function of {{ plugin }} has prevented it
+    {% endif %} from loading in project {{ team }}. Here's what happened:
 </p>
 <pre><code>{{ error }}</code></pre>
-<p>The app has been disabled automatically. Check its logs and re-enable it from the Apps page of PostHog.</p>
+<p>Don't worry, we've disabled the app for now. If this it the first time you've seen the error, we recommend checking the logs and re-enabling it from the Apps page.</p>
 <div class="mb mt text-center">
     <a class="button" href="{% absolute_uri '/project/apps' %}">Go to Apps</a>
 </div>
 <p>
-    If you find that you're unable to resolve the issue, contact the author of the app or the PostHog team.
+    If the issue persists, please check the {{ plugin }} <a href="https://posthog.com/docs/apps">docs for more help</a>. 
```

Comment:
This might be a bit misleading because it will say "<app name> docs" but the link points to general Apps docs

Reply:
Hm. Is there was a way to direct to the relevant app docs? Or should we just commit this suggestion?

```suggestion
    If the issue persists, please <a href="https://posthog.com/docs/apps">check the app docs for more help</a>. 
```

Reply:
To be honest https://posthog.com/docs/apps just doesn't seem helpful if you get a fatal app error. Any such issue is very app-specific, so I think the original guidance of contacting "the author of the app or the PostHog team" is going to be the most informative.
For most plugins a good pointer would be `plugin.url`, which usually points to the GitHub repo of the plugin (where an issue can be created), but not all have that.

Reply:
Yeah, my issue with the previous wording is that the immediate question is then: Ok, who is the author of the app and where can I find that out? Which is information we document in the `/docs/apps/` listing for each app. 

Doesn't sound like there's a better solution than this though. 

```suggestion
    If the issue persists, please contact the author of the app, or the PostHog team, for help.
```

Reply:
No a quick solution available right now :(